crawler.py

```diff
@@ -1,32 +1,43 @@
+import requests
+
 from bs4 import BeautifulSoup
 
-import requests
 
-def parse():
```

Comment:
It's pretty important to give your variables/functions/classes clean, proper and not misguiding names.

Your function crawls over the site (_and_ does content parsing), so `crawl()` looks a bit better name in this case.